src/components/Form.js

```diff
@@ -0,0 +1,52 @@
+import { useState, useRef } from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+
+export default function Form({ savedList, setSavedList }) {
+  const [input, setInput] = useState([]);
+  const onChange = (e) => {
+    setInput(e.target.value);
+  };
+
+  const nextId = useRef(1);
+
+  const onCreate = (e) => {
+    e.preventDefault();
+    const toDo = { id: uuidv4(), content: input, isDone: false };
+    const updatedList = [...savedList, toDo];
+    setSavedList(updatedList);
+    setInput('');
+    nextId.current += 1;
+  };
+
+  return (
+    <form>
+      <InputContainer
+        type={'text'}
```

Comment:
styled-component의 컴포넌트도 결국 원래 태그의 속성을 그대로 상속받기 때문에 이렇게 적어도 무방합니다.
```suggestion
        type='text'
```

Reply:
알려주셔서 감사합니다!! 정성스러운 코멘트도 감사드려요🙌